presto-native-execution/pom.xml

```diff
@@ -299,5 +326,86 @@
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <version>3.3.1</version>
+                        <executions>
+                            <execution>
+                                <id>copy-presto-cli-executable</id>
+                                <phase>prepare-package</phase>
+                                <goals>
+                                    <goal>copy-resources</goal>
+                                </goals>
+                                <configuration>
+                                    <outputDirectory>${project.parent.basedir}/docker/</outputDirectory>
+                                    <resources>
+                                        <resource>
+                                            <directory>${project.parent.basedir}/presto-cli/target</directory>
+                                            <includes>
+                                                <include>presto-cli-*-executable.jar</include>
+                                            </includes>
+                                        </resource>
+                                        <resource>
+                                            <directory>${project.parent.basedir}/presto-server/target</directory>
+                                            <includes>
+                                                <include>presto-server-*.tar.gz</include>
+                                            </includes>
+                                        </resource>
+                                    </resources>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>com.spotify</groupId>
+                        <artifactId>dockerfile-maven-plugin</artifactId>
+                        <version>1.4.13</version>
+                        <executions>
+                            <execution>
+                                <id>coordinator</id>
+                                <phase>install</phase>
+                                <goals>
+                                    <goal>build</goal>
+                                </goals>
+                                <configuration>
+                                    <repository>presto-coordinator</repository>
+                                    <tag>latest</tag>
+                                    <contextDirectory>${project.parent.basedir}/docker/</contextDirectory>
+                                    <dockerfile>${project.parent.basedir}/docker/Dockerfile</dockerfile>
+                                    <buildArgs>
+                                        <PRESTO_VERSION>0.289-SNAPSHOT</PRESTO_VERSION>
```

Comment:
Does this VERSION need to be hard-coded ?

Reply:
Can't you use `${project.version}`?

Reply:
used `${project.version}`, didn't knew this!